pkgs/development/libraries/spdk/default.nix

```diff
@@ -10,9 +10,19 @@
 , libuuid
 , numactl
 , openssl
+, fetchurl
 }:
 
-stdenv.mkDerivation rec {
+let
+  # The old version has some CVEs howver they should not affect SPDK's usage of the framework: https://github.com/NixOS/nixpkgs/pull/171648#issuecomment-1121964568
+  dpdk' = dpdk.overrideAttrs (old: rec {
```

Comment:
if we're doing this, why don't we just use the git submodule they include for dpdk?

Reply:
I agree with Mic92's approach, since it would introduce a lot of extra complexity and transitive dependencies to the spdk expression for what is hopefully only a temporary hack. One thing I might suggest though is using specifically spdk's dpdk fork rather than upstream dpdk? Not familiar enough with the domain to say if that makes sense or not though.

Reply:
> One thing I might suggest though is using specifically spdk's dpdk fork rather than upstream dpdk?

That's what I meant

Reply:
Doesn't appear to be trivial (I've given it a try and run into two problems already), so as long as upstream dpdk works I'd say we should just use that.